hydra/_internal/hydra.py

```diff
@@ -188,19 +189,22 @@ def show_cfg(
         )
         if package == "_global_":
             package = None
-        if package is not None:
+
+        if package is None:
+            ret = cfg
+        else:
             ret = OmegaConf.select(cfg, package)
             if ret is None:
                 sys.stderr.write(f"package '{package}' not found in config\n")
                 sys.exit(1)
-            else:
-                if isinstance(ret, Container):
-                    print(f"# @package {package}")
-                    sys.stdout.write(OmegaConf.to_yaml(ret))
-                else:
-                    print(ret)
-        else:
-            sys.stdout.write(OmegaConf.to_yaml(cfg))
+            if not isinstance(ret, Container):
+                print(ret)
+                return
+            print(f"# @package {package}")
+
+        if resolve:
+            OmegaConf.resolve(ret)
+        sys.stdout.write(OmegaConf.to_yaml(ret))
```

Comment:
```suggestion
        sys.stdout.write(OmegaConf.to_yaml(ret, resolve=resolve))
```

Reply:
Gave this a try just now (commit https://github.com/facebookresearch/hydra/pull/1616/commits/bfc702a65af56dd0d5f7ced7b55e09e0b214e23c) but the build failed.
The issue was that `OmegaConf.to_yaml(..., resolve=True)` raises `MissingMandatoryValue` if there are interpolations pointing to missing keys, while `OmegaConf.resolve(...)` simply replaces the interpolation-to-missing with `"???"`.

Reply:
gotcha. we can keep it then.